INSTALL

```diff
@@ -969,7 +969,7 @@
 
  BUILDFILE
                 Use a different build file name than the platform default
-                ("Makefile" on Unixly platforms, "makefile" on native Windows,
+                ("Makefile" on Unix-like platforms, "makefile" on native Windows,
```

Comment:
Seems like it's a pet term of @levitte (bb09fd2bb6a ed43fe73ea3 207c7df746c). But since he approved, well let's remove it. But also in this case there are a lot more locations

```make
crypto/LPdir_unix.c:54: * The POSIXly macro for the maximum number of characters in a file path is
Configurations/00-base-templates.conf:48:	#### a BASE and assume Unixly defaults
e_os.h:248:     POSIXly.  'man perlvms' explains it further.
INSTALL:972:                ("Makefile" on Unixly platforms, "makefile" on native Windows,
INSTALL:1174: On most POSIXly platforms, shared libraries are named libcrypto.so.1.1
apps/ca.c:608:         * directory syntax to Unixly, and give that to app_isdir,
```

Reply:
It *is* a pet term, and I *am* sad to see it go...  but code is increasingly humourless (see the well known controversy with tuning a fish), so I've resigned myself to that...

Reply:
It's an old tunefs joke, which is in turn a travesty of a song from the 70's.

http://catb.org/jargon//html/T/tunafish.html
https://en.wikipedia.org/wiki/You_Can_Tune_a_Piano,_but_You_Can%27t_Tuna_Fish

(I'm obviously showing my age here ;-))

Reply:
@levitte Thank you for sharing the tunefs reference, that most certainly gave me a laugh!